packages/suite-desktop/package.json

```diff
@@ -100,7 +100,14 @@
                     "from": "build/static/bridge",
                     "to": "bridge",
                     "filter": [
-                        "**/trezord-win.exe"
+                        "**/trezord-win-x32.exe"
```

Comment:
it should be enough to add "**/trezord-win-x64.exe" here as a second item of an array, (or use regexp `**/trezord-win-*.exe`? not sure)

Reply:
I'll add the second item of an array. I don't want to use `*`, because eventually we need to split these binaries into various installers, not to install all binaries with one installer.